CHANGELOG.MD

```diff
@@ -0,0 +1,43 @@
+# Change Log
+## 1.1 Release
+	- Supports Houdini 18.0 and USD 19.11
+	- Uses RPR SDK 1.34.5
+	- Uses RIF 1.4.3
+### New features
+	- Added Centos 7 build
+	- Added RPR Render Settings node for use in Houdini in addition to global per project settings.
```

Comment:
Actually, all render settings that were available globally in "Display Options" moved to native LOP node "Render Settings" into its own tab "RPR", so there are no render settings in `global per project settings`